src/DOM/hydration.ts

```diff
@@ -5,6 +5,7 @@ import {
 	replaceChild,
 } from './utils';
 import {
+	EMPTY_OBJ,
```

Comment:
May sound strange but rather than import like this, it's better to do:
`import { EMPTY_OBJ } from 'inferno'`